leggen/main.py

```diff
@@ -87,13 +87,21 @@ def get_command(self, ctx, name):
     show_envvar=True,
     help="Path to TOML configuration file",
 )
+@click.option(
+    "--api-url",
+    type=str,
+    default=None,
+    envvar="LEGGEND_API_URL",
+    show_envvar=True,
+    help="URL of the leggend API service (default: http://localhost:8000)",
+)
```

Comment:
The default is None even though the help states that the default is http://localhost:8000